### ISSUE TITLE
Coloring for angle bracket doesn't work in C++ code

### ISSUE THREAD

**Issue by hwooo** [izhangzhihao/intellij-rainbow-brackets#227]
**“Star” this repo before you creating a issue, thanks for your support 😁**

Your issue may already be reported!
Please search on the [issue track](https://github.com/izhangzhihao/intellij-rainbow-brackets/issues) before creating one.

## Your programming language(s)
C++

## Expected Behavior
Different color of nested angle brackets(`<>`)

## Current Behavior
Coloring for angle bracket doesn't work

## Code snippet for reproduce (for bugs)
![Capture](https://user-images.githubusercontent.com/46282492/58976871-6f97a780-8803-11e9-8600-0ec2b816d35d.PNG)
`std::vector<std::vector<std::string>> ret`

## Your Environment

* Plugin version: 5.19
* IDE & Operating System version, comment your env as below(go to "About IntelliJ IDEA" -> click the "copy" icon):

```
CLion 2019.1.4
Build #CL-191.7479.33, built on May 30, 2019
JRE: 1.8.0_202-release-1483-b53 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
```

**Comment by izhangzhihao** [comment]
![image](https://user-images.githubusercontent.com/12044174/79865668-1b42e980-840e-11ea-8d57-46b9c01de187.png)

**Comment by izhangzhihao** [comment]
This feature will happen in the next release. If you want to have a try, please download the snapshot build from [here](https://izhangzhihao.github.io/rainbow-brackets-document/#install-snapshot-build). By the way, please considers donating to this project on [Open Collective](https://opencollective.com/intellij-rainbow-brackets) if you love this plugin.

**Comment by halfelf** [comment]
Has this issue been fixed ?  I'm using 6.11 and still cannot see any **nested** angle brackets colorized (like the red underlined line in your screenshot). Other single pairs of angle brackets work.

**Comment by izhangzhihao** [comment]
For nested angle brackets, it's not possible for now, because the nested brackets are not correctly parsed by the IDE.
